test/conformance/testdata/conformance.yaml

```diff
@@ -2843,6 +2843,196 @@
     lifecycle of a container.
   release: v1.19
   file: test/e2e/common/node/expansion.go
+- testname: 'DRA driver: creates slices'
+  codename: '[sig-node] [DRA] [FeatureGate:DynamicResourceAllocation] ResourceSlice
+    Controller creates slices [Conformance]'
+  description: The helper code for DRA drivers must be able to publish large ResourceSlices.
+  release: v1.34
+  file: test/e2e/dra/dra.go
+- testname: 'Dynamic Resource Allocation: must apply per-node permission checks'
+  codename: '[sig-node] [DRA] [FeatureGate:DynamicResourceAllocation] control plane
```

Comment:
We should probably drop all the `FeatureGate` from conformance tests. Conformance tests may not be skipped.

(This is new, we wound up doing that in https://github.com/kubernetes/kubernetes/pull/132219 and I think that's correct)

Reply:
Removal is pending in https://github.com/kubernetes/kubernetes/pull/132706.